qrenderdoc/Code/QRDUtils.cpp

```diff
@@ -442,6 +442,29 @@ QMessageBox::StandardButton RDDialog::messageBox(QMessageBox::Icon icon, QWidget
   return ret;
 }
 
+QMessageBox::StandardButton RDDialog::messageBoxChecked(QMessageBox::Icon icon, QWidget *parent,
+                                                        const QString &title, const QString &text,
+                                                        QCheckBox *checkBox, bool &checked,
+                                                        QMessageBox::StandardButtons buttons,
+                                                        QMessageBox::StandardButton defaultButton)
+{
+  bool isChecked = checked;
```

Comment:
No, I think that is leftover from some debugging attempts.  Updated to use checked directly.